scripts/install.sh

```diff
@@ -76,19 +78,19 @@ shift $(expr $OPTIND - 1) # remove options from positional parameters
 
 # Overwrite default properties
 NETWORK=${network:-$NETWORK_DEFAULT}
-TBTC_PATH=${tbtc_path:-""}
-THRESHOLD_NETWORK_PATH=${threshold_network_path:-""}
+TBTC_PATH=${tbtc_path:-"$LOCAL_TBTC_SOL_PATH"}
+THRESHOLD_NETWORK_PATH=${threshold_network_path:-"$LOCAL_THRESHOLD_NETWORK_PATH"}
```

Comment:
Let's leave them empty in case the user sets flags to the tmp directory. In such scenario we want to use what's already in the directory, not overwrite it with fresh clone.

```suggestion
TBTC_PATH=${tbtc_path:-""}
THRESHOLD_PATH=${threshold_network_path:-""}
```

Reply:
Even if a user sets a flag to the `tmp` directory, that directory will be empty already. We clean the `tmp` directory every time we run `install.sh` for fresh installation to ensure nothing is cached or misconfigured.